turbopack/crates/turbo-tasks-backend/src/backend/storage.rs

```diff
@@ -1,9 +1,12 @@
 use std::{
     hash::Hash,
     ops::{Deref, DerefMut},
+    sync::{atomic::AtomicBool, Arc},
     thread::available_parallelism,
 };
 
+use bitfield::*;
```

Comment:
Do we really need to use a wildcard import? I don't like wildcard imports because their behavior can change when the upstream library adds new exports. https://www.lurklurk.org/effective-rust/wildcard.html

Reply:
Yeah, not sure how I ended up with this import...